sound/soc/intel/boards/sof_da7219.c

```diff
@@ -395,10 +539,16 @@ static int audio_probe(struct platform_device *pdev)
 static const struct platform_device_id board_ids[] = {
 	{
 		.name = "sof_da7219_mx98373",
-		.driver_data = (kernel_ulong_t)(SOF_MAX98373_SPEAKER_AMP_PRESENT),
+		.driver_data = (kernel_ulong_t)(SOF_MAX98373_SPEAKER_AMP_PRESENT |
+					SOF_DA7219_JSL_BOARD),
 	},
 	{
 		.name = "sof_da7219_mx98360a",
+		.driver_data = (kernel_ulong_t)(SOF_MAX98360A_SPEAKER_AMP_PRESENT |
+					SOF_DA7219_JSL_BOARD),
+	},
+	{
+		.name = "adl_mx98360_da7219",
```

Comment:
is there a reason why you dropped the 'sof' prefix?

Reply:
You mean the driver name "adl_mx98360_da7219"? I just follow the convention used in other machine drivers.

Two names "sof_da7219_mx98373" and "sof_da7219_mx98360a" were created for JSL boards long time ago so I didn't touch the names. Do you want me to change them, too?

Reply:
that would be a nice clean-up, I am not sure what we have those two drivers that are named differently?

````
soc-acpi-intel-jsl-match.c\055:		.drv_name = "sof_da7219_mx98373",
soc-acpi-intel-jsl-match.c\062:		.drv_name = "sof_da7219_mx98360a",
````

Reply:
driver names changed